thermalnetwork/tests/test_network.py

```diff
@@ -129,18 +129,6 @@ def test_network_one_ghe_detailed_geometry(self):
 
         assert res == 0
 
-        updated_sys_param = load_json(self.sys_param_path_1_ghe_detailed_geometry)
-
-        expected_hydraulic_dia = 0.1377
-        expected_pump_head = 128704
-        expected_flow_rate = 0.018
-
-        self.check_horiz_pipe_params(updated_sys_param, expected_hydraulic_dia)
-        self.check_pump_params(updated_sys_param, expected_pump_head, expected_flow_rate)
-
-        expected_ghe_data = {"8c369df2-18e9-439a-8c25-875851c5aaf0": {"num_bh": 50, "length": 89.5}}
-        self.check_ghe_data(updated_sys_param, expected_ghe_data)
-
```

Comment:
Understood that things might have moved. IMO, it's still valuable to at least pin it to the current performance so we can know if it has changed, and if so, by how much. It just helps gauge whether any potential changes are reasonable, or if something larger is occurring.

Reply:
I updated all parts of the test.  So this one should be good.